src/test/scala/ai/eto/rikai/sql/spark/execution/CreateModelCommandTest.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 Rikai authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.eto.rikai.sql.spark.execution
+
+import ai.eto.rikai.SparkTestSession
+import ai.eto.rikai.sql.model.Catalog
+import org.scalatest.funsuite.AnyFunSuite
+
+class CreateModelCommandTest extends AnyFunSuite with SparkTestSession {
+
+  test("create model from uri") {
+
+    spark
+      .sql("CREATE MODEL model_created USING 'fake://model/created/from/uri'")
+      .count()
+    assert(Catalog.testing.modelExists("model_created"))
+
+    val model = Catalog.testing.getModel("model_created").get
+    assert(model.name == "model_created")
+    assert(model.uri == "fake://model/created/from/uri")
+    assert(model.options.isEmpty)
+  }
+
+  test("create model with options") {
+    spark
+      .sql(
+        "CREATE MODEL model_options OPTIONS (foo='bar',num=1.2,flag=True) USING 'fake://foo'"
```

Comment:
For booleans, can they be `"true"`(as string)?

Reply:
it will make things more confused for user.